atomic_reactor/util.py

```diff
@@ -541,6 +542,21 @@ def get_image_upload_filename(metadata, image_id, platform):
                            platform=platform, ext=ext)
 
 
+def get_output_metadata(path, filename):
+    """
+    Describe a file by its metadata.
+
+    :return: dict
+    """
+    checksums = get_checksums(path, ['md5'])
+    metadata = {'filename': filename,
+                'filesize': os.path.getsize(path),
+                'checksum': checksums['md5sum'],
+                'checksum_type': 'md5'}
+
+    return metadata
```

Comment:
Minor: I wonder if this, and the other functions added to this file, would be better in koji_util.py -- they're only used for Koji metadata I think.

Reply:
sure I can move all added from util to koji_util